scripts/generate_genotype_df/main.py

```diff
@@ -0,0 +1,23 @@
+"""Entry point for the analysis runner."""
+
+import os
+import hailtop.batch as hb
+from analysis_runner import dataproc
+
+service_backend = hb.ServiceBackend(
+    billing_project=os.getenv('HAIL_BILLING_PROJECT'), bucket=os.getenv('HAIL_BUCKET')
+)
+
+batch = hb.Batch(name='export_genotype_data', backend=service_backend)
+
+dataproc.hail_dataproc_job(
+    batch,
+    f'generate_genotype_df.py',
+    max_age='4h',
+    num_secondary_workers=20,
```

Comment:
Do you need that many workers and the disk override or is this just a remnant from a previous script?

Reply:
This was an old remnant- thanks for pointing it out!